apps/content/migrator/migrations/6/index.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require('mongoose')
+const { appConfiguration } = require('../../../config')
+const TENANT = process.env.TENANT || '0'
+
+const Configuration = mongoose.model('Configuration')
+
+/**
+ * update site configuration
+ */
+
+/**
+ * check if app-configuration doesn't have themeStylesUrl propery
+ */
+async function check () {
+  const hasMissingThemeConfig = await Configuration.countDocuments({ key: appConfiguration, 'metadata.themeStylesUrl': {$exists: false} })
+  return !hasMissingThemeConfig
+}
+
+/**
+ * migrate relevant db rows to fit the new upgrade
+ */
+function migrate () {
+  console.log('start appConfiguration.metadata.themeStylesUrl migration')
+  return Configuration.aggregate([
+    { $match: { 'metadata.themeStylesUrl': {$exists: false} } },
+    { $limit: 20 },
+    { $project: { _id: 1 } }
+  ])
+    .then(async (list) => {
+      if (!list || !list.length) {
```

Comment:
simplify` if (!list?.length)`

Reply:
Should run on node 14 vanilla.

Reply:
Why? Don't we always talk about shifting to the newer, stable versions of node, while simultaneously replacing existing code with typescript equivalent?

Reply:
we need to start working on the upgrade for node 15+, but currently, this migration should work on existing environments. :)